docs/advanced/modules.md

```diff
@@ -0,0 +1,46 @@
+# Modules
+
+Modules are subapplications that are run as application plugins.
+They're used to define pieces of functionality that are pluggable to Decidim.
+
+Decidim's modules are no more than Ruby on Rails engines that should be required in the application's `Gemfile`.
+
+## Example
+
+A typical engine looks like the following:
+
+```ruby
+module Decidim
+  module Verifications
+    module Sms
+      # This is an engine that authorizes users by sending them a code through an SMS.
+      class Engine < ::Rails::Engine
+        isolate_namespace Decidim::Verifications::Sms
+
+        paths["db/migrate"] = nil
```

Comment:
@mrcasals @oriolgual @josepjaume , do you know why are we removing the `db/migrate` path? Is it to avoid to execute migrations inside modules when running migrations in the application?